pallets/verifiers/src/lib.rs

```diff
@@ -137,8 +146,27 @@ pub mod pallet {
             + IsType<<Self as frame_system::Config>::RuntimeEvent>;
```

Comment:
I was thinking, shall we enforce a MAX_VK_SIZE (s.t. people are not able to register vks bigger than this) ? Perhaps it's safer (also the preimage pallet does that). We should discuss whether it's better this to be in the config of the pallet or an hardcoded constant somewhere

Reply:
At the moment every verifier pallet is already imposing a limit on the size of the vk, so it's already impossible to register arbitrarily big vks. If we deem it necessary to introduce a global common limit, I would propose to do so in a different PR.